src/tools/gophish_export.py

```diff
@@ -400,6 +401,14 @@ def main() -> None:
             logging.critical(e.args[0])
             sys.exit(1)
 
+    if not validate_assessment_id(args["ASSESSMENT_ID"]):
+        logging.critical(
+            '"%s" is an invalid assessment_id format. Assessment Identifiers begin with RV and followed by a 4 or 5 '
+            "digit numerical sequence. Example: RV1234",
```

Comment:
Small capitalization/grammar cleanup:
```suggestion
            '"%s" is an invalid assessment_id format. Assessment identifiers begin with RV and are followed by '
            " a 4 or 5 digit numerical sequence. Examples: RV1234, RV12345",
```

Reply:
See here for changes applied 

9c83ccb